model-optimizer/extensions/front/tf/CTCLoss_ext.py

```diff
@@ -23,11 +23,18 @@ class CTCLossFrontExtractor(FrontExtractorOp):
 
     @classmethod
     def extract(cls, node):
+        # For CTCLoss default value is [N, T]
+        logits_time_major = True
+        if 'logits_time_major' in node.pb.attr:
+            logits_time_major = not node.pb.attr['logits_time_major'].b
```

Comment:
why `not node.pb.attr['logits_time_major'].b`?

Reply:
It mistakes. Not was removed.